### ISSUE TITLE
Bug on update an Action using PostgreSQL datastore plugin

### ISSUE THREAD

**Issue by douglaszuqueto** [tago-io/tcore#9]
Bug on update an Action using PostgreSQL datastore plugin

Context:

* TCore: V0.5.0
* Datastore Plugin: PostgreSQL V0.5.0
* Database: PostgreSQL 14
* Module: Create Action

### Request:

```json
{
  "active": true,
  "name": "demo",
  "tags": [],
  "type": "condition",
  "trigger": [
    {
      "variable": "temperature",
      "is": "=",
      "value": "30",
      "value_type": "number",
      "device": "613ab8e78e9c5be5e1d6035a"
    }
  ],
  "lock": false,
  "action": {
    "type": "script",
    "script": [
      "613aa2994311091386e882ce"
    ]
  }
}
```
### Response:

Error: 400 Bad Request

```json
{
  "status": false,
  "message": "update \"action\" set \"action\" = $1, \"active\" = $2, \"name\" = $3, \"tags\" = $4, \"trigger\" = $5, \"type\" = $6, \"lock\" = $7 where \"id\" = $8 - invalid input syntax for type json"
}
```

### Screenshot

![Screenshot](https://user-images.githubusercontent.com/7410247/180490567-8647be05-9e71-4dc4-96e6-9f34db4c0bde.png)

**Comment by douglaszuqueto** [comment]
I changed field type of trigger from **_json_** to **_jsonb[]_** in the table and i have successful.

```sql
CREATE TABLE IF NOT EXISTS tcore.action
(
    id character varying(24) COLLATE pg_catalog."default" NOT NULL,
    name character varying(100) COLLATE pg_catalog."default",
    active boolean,
    tags jsonb[],
    action json,
    device_info json,
    last_triggered timestamp with time zone,
    created_at timestamp with time zone,
    updated_at timestamp with time zone,
    type text COLLATE pg_catalog."default",
    lock boolean,
    trigger jsonb[] # here,
    CONSTRAINT action_pkey PRIMARY KEY (id)
)
```

**Comment by requeijaum** [comment]
I'm using TagoCore v0.4.1 running on RPi 4B arm64 (Ubuntu 22.04) and having a similar bug in the Action list.

Instead of PostgreSQL, I'm using MySQL database plugin.

![image](https://user-images.githubusercontent.com/5564635/180615401-fa9e9af5-9ac3-427f-8cc1-c73b0919de5b.png)
![image](https://user-images.githubusercontent.com/5564635/180615405-e0670b92-7f6d-4b60-a7fe-5788df7e1eb6.png)
![image](https://user-images.githubusercontent.com/5564635/180615416-23d5dfdf-ecdc-46d0-bae4-301e5be75ca6.png)
![image](https://user-images.githubusercontent.com/5564635/180615420-0b890f11-245b-4f0d-afb5-a0aaf166da16.png)

I did a quick fix accessing the database and changing the 'lock' column from 'tcore.action' to tinyint(1) instead of string or boolean. I created a new action, changed the lock state to 'null' and got the Action's list working.
